src/MongoDB.Driver.Legacy/Builders/IndexKeysBuilder.cs

```diff
@@ -261,6 +271,30 @@ public override BsonDocument ToBsonDocument()
             return _document;
         }
 
+        /// <summary>
+        /// Sets a wildcard key to the index. If the wildcard name is empty, the generated key will be for the whole collection.
+        /// </summary>
+        /// <param name="name">The wildcard key.</param>
+        /// <returns>The builder (so method calls can be chained).</returns>
+        public IndexKeysBuilder Wildcard(string name = null)
+        {
+            string wildcard = name;
+            if (string.IsNullOrWhiteSpace(name))
```

Comment:
I think an empty string is an error.

Reply:
I removed this condition since currently, we implement this logic via the default value.
Done.